js/script.js

```diff
@@ -0,0 +1,134 @@
+/*  Anna Sullivan
+    script.js
+    INFO2134WW
+    Thoendel
+    5/2/2020
+*/
+window.addEventListener('load', () => {//event listener on page load
+    
+    startClock();
+        //YOUR CODE SHOULD START BELOW THIS LINE
+    
+    navigator.geolocation.getCurrentPosition(findLocation);//call getCurrentPosition using navigator and geolocation
+        
+    function findLocation(position){//function to find position of user
+       console.log(`Latitude: `+ position.coords.latitude `, Longitude: `+ 
+       position.coords.longitude +`.`);
+    }
+    
+    const buttons = document.getElementsByTagName('button'); //const set to the button tag name
+    
+    for(const button of buttons){//for loop button const
+        const buttonID = button.dataset.id; //const buttonID set to button id
+    button.addEventListener('click', () => {//event listen on click of button
+        if(buttonID == 'getCurrentWX'){//if buttonID = getCurrentWX call function getCurrentWX

```

Comment:
the data-id is getCurrentWx (not WX) in your index.html. I would make it the id and not the data-id - so you could use the event.target to differentiate. You can't know which button was clicked before it was clicked. So line 22 is pretty irrelevant. Here you would want something more like:
button.addEventListener('click', (evt) => {
if (evt.target.id == 'getCurrentWX') => { do stuff!

Make sense?